examples/dem/3d-bouncing-particle/bouncing_particle_original.tpl

```diff
@@ -12,7 +12,7 @@ subsection simulation control
   set time end          = 8
   set log frequency     = 100000
   set output frequency  = 100000
-  set output path       = ./out_{{ER}}/
+  set output path       = ./bouncing_particle_{{ER}}/
```

Comment:
I'm not sure i like this change because removing the folders become more annoying after that. You cannot do rm -r bouncing_particle_* ...

Reply:
Ok, I will fix this